final-project/core/backend/app/routers/kanji.py

```
@@ -0,0 +1,123 @@
+    offset = (page - 1) * ITEMS_PER_PAGE
+    
+    # Get total count
+    result = await db.execute(select(func.count()).select_from(Kanji))
+    total_count = result.scalar()
+    
+    # Get kanji with their stats
+    query = (
+        select(
+            Kanji,
+            func.count(ReviewItem.id).label("total_reviews"),
+            func.sum(case((ReviewItem.correct == True, 1), else_=0)).label("correct_reviews")
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if ReviewItem.correct:` for truth checks

Replace with `ReviewItem.correct`

(E712)